ocaml/typing/layouts.ml

```diff
@@ -175,6 +175,26 @@ module Sort = struct
 
     and var ppf v = fprintf ppf "{ contents = %a }" opt_t (!v)
   end
+
+  let sort_function = value
```

Comment:
Somehow `Sort.sort_function` feels a lot clearer to me than `Sort.function`, but I'm open to disagreement.  (And in this particular case, because `function` is a keyword, it would have to be `function_` or something).

Reply:
I disagree that the repetition helps, but I don't feel strongly. Perhaps `Sort.for_function` as in "the sort for functions" would be best, but I still don't feel strongly.

Reply:
I've changed it to `for`.  Something about having `Sort.value` and `Sort.function` look so similar seems odd to me - surely we'll want `Sort.function` for type constructors later ;)